extension/data/styles/tbui.css

```
@@ -96,13 +96,40 @@
 
 /* Footer stuff */
 .mod-toolbox-rd .tb-window-footer {
-    width: 100%;
-    height: 36px;
     display: flex;
+    gap: 5px;
+    padding: 5px;
     justify-content: center;
     align-items: center;
     background-color: #CEE3F8;
 }
+/* A form can be the only child of the footer - use it as the flex container */
+.mod-toolbox-rd .tb-window-footer > form {
```

If this rule is specifically for the form being the only child, we may want to do
```suggestion
.mod-toolbox-rd .tb-window-footer > form:only-child {
```

---

I don't think this is really necessary tbh - there's no case where a `form` has
siblings in a footer, and I can't think of a reason we'd need to do that in the
future either. Plus, since these rules set the `gap` and other flex layout stuff
consistently between the `form` and the footer itself, I think this should work
pretty much as-is in the case where there *is* a `form` with arbitrary siblings,
so there's no need to arbitrarily lock this down.